### ISSUE TITLE
[MAS2.4.4][Screen reader-Welcome] “Locally” link is not descriptive present in the welcome tab screen 

### ISSUE THREAD

**Issue by awalia13** [microsoft/BotFramework-Emulator#1714]
**Actual Result:**
“Locally, Open bot” link is not descriptive present in the welcome tab screen 

**Expected Result:**
“Locally, Open bot ” link should be more descriptive and should be meaning full aria-label text.

Repro Steps:
Turn on the narrator 
1.Open the Bot Framework V4 Emulator.
2.Navigate to the Welcome tab screen a
3.Navigate to the “Locally” link 
4.Observe the issue 

User Impact:
Users who rely on Screen Readers will be affected to navigate the links if Narrator is not narrating the descriptive info for link.

Test Environment:
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator 
Screen reader: Narrator 

MAS Reference Link:
[Sharepoint Link](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7})
![MAS2 4 4_Welcome](https://user-images.githubusercontent.com/46575396/62776229-91ddc800-bac7-11e9-8267-3e6dc7f711f0.png)
[MAS2.4.4_Welcome.zip](https://github.com/microsoft/BotFramework-Emulator/files/3486138/MAS2.4.4_Welcome.zip)

**Comment by tonyanziano** [comment]
Fixed by #1775

**Comment by awalia13** [comment]
@tonyanziano  - Please don't close the bugs. If the bug is fixed then please add a tag "Fixed" so that we came to know it has been fixed and ready to validate. We will validate and closed it if it is working fine.

**Comment by tonyanziano** [comment]
@awalia13 Ok, I will add a "Fixed" label once a nightly build is out with the changes. You can download the newest nightly builds from [this repository](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/02.echo-bot).
